README.md

```diff
@@ -344,8 +344,38 @@ For a comparison of some of these, see this
 
 ## Storage Backends
 
-In order to not download or scan any previously scanned sources again, the _scanner_ can use a storage backend to store
-scan results for later reuse.
+In order to not download or scan any previously scanned sources again, or to reuse scan results generated via other
+services, the _scanner_ can be configured to use so-called storage backends. Before processing a package, it checks
+whether compatible scan results are already available in one of the storages declared; if this is the case, they
+are downloaded and reused. Otherwise, the package's source code is fetched and scanned. Afterwards, the new scan
```

Comment:
Nit: I think we should swap the "downloaded" and "fetched" word in this sentence. Rationale: The second action is actually performed by ORT's _downloader_, hence the verb "download" makes sense. And just to not repeat the word "download" so close by, I'd use "fetched" in the first occurrence.